sessions/slides/improving-forecasting-models.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Improving Forecasting Models"
+author: "Nowcasting and forecasting of infectious disease dynamics"
+engine: knitr
+format:
+  revealjs:
+    output: slides/improving-forecasting-models.html
+    footer: "Improving Forecasting Models"
+    slide-level: 3
+---
+
+### Which Model to Use?
+
+There are *many* types of models available to us, (you've already seen a lot now):
+
+-   A range of choices from mechanistic to purely statistical.
+
+-   Bayesian, frequentist, machine learning.
+
+-   Stochastic vs. Deterministic
+
+How should we decide which model(s) to incorporate for nowcasting and forecasting? 
+
+What outputs and features do we require of these models?
+
+
+### Using Data the Data to Help Inform your Modeling Choices
```

Comment:
Could we add another slide here showing the data plotted where each season is overlaid with the others? This will help emphasize and motivate discussions about seasonality.